pkgs/development/libraries/mlt/default.nix

```diff
@@ -94,7 +100,7 @@ stdenv.mkDerivation rec {
     SDL
   ] ++ lib.optionals enableSDL2 [
     SDL2
-  ];
+  ] ++ lib.optionals enableGlaxnimate glaxnimate.buildInputs;
```

Comment:
I think this doesn't quite work as intended anymore. Maybe the issue is that this unconditionally adds `qtbase` and other qt packages, but `mlt` can only deal with them when `qt != null` in the inputs. Maybe throw an error if `enableGlaxnimate = true && qt == null`?